lib/shelf_web_socket.dart

```diff
@@ -38,7 +38,9 @@ typedef _BinaryFunction = void Function(Null, Null);
 ///
 /// [origin considerations]: https://tools.ietf.org/html/rfc6455#section-10.2
 Handler webSocketHandler(Function onConnection,
-    {Iterable<String> protocols, Iterable<String> allowedOrigins}) {
+    {Iterable<String> protocols,
+    Iterable<String> allowedOrigins,
+    Duration pingInterval}) {
```

Comment:
Please add a note about the new argument to the doc comment.

Reply:
Added!